idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/optics/OpticsIdePlugin.kt

```diff
@@ -18,3 +23,10 @@ val IdeMetaPlugin.opticsIdePlugin: Plugin
       )
     )
   }
+
+class OpticsIdePlugin : LineMarkerProvider, LineMarkerSyntax {
```

Comment:
Even though we register an Intention with vanilla `intelliJ`, the optics tests still fail. You may test it with an example project and this code:
```
data class TestLenses(val a: String, val b: String)

data class Person(
    val name: String,
    val age: Int,
    val home: Home
)

data class Home(val street: String)
```